scripts/ui_release.sh

```diff
@@ -13,7 +13,7 @@ publish_workspaces=$(for workspace in ${workspaces}; do
   # packages are private so we shouldn't try to publish them.
   # TODO: see if we can do something smarter here by checking for "private" set
   # to true in a given package's package.json.
-  if [[ "${workspace}" != "app" ]] && [[ "${workspace}" != "e2e" ]] && [[ "${workspace}" != "storybook" ]]; then
+  if [[ "${workspace}" != "app" ]] && [[ "${workspace}" != "e2e" ]] && [[ "${workspace}" != "storybook" ]]; && [[ "${workspace}" != "internal-utilities" ]]; then
```

Comment:
I feel bad every time I add another check in here, but I don't know a quick way to do this in a more clever way in bash. 😢

Reply:
Yeahh looks fine for now, not like this gets called a ton or anything